From d0e9432dcf77fcc89f9e6c78bc755b2bd984a6d1 Mon Sep 17 00:00:00 2001
From: Chen Meng J <meng.j.chen@intel.com>
Date: Tue, 4 Jun 2019 13:23:37 +0800
Subject: [PATCH 1167/1225] media: intel-ipu4: Fix possible deadlock

unified naming in function ipu_buttress_add_psys_constraint().

Change-Id: Idd1dde2d49a6f2e5d896685e1f9b95c557e59549
Tracked-On: PKT-2588
Tracked-On: #H1507238979
Signed-off-by: Chen Meng J <meng.j.chen@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/pci/intel/ipu-buttress.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/pci/intel/ipu-buttress.c b/drivers/media/pci/intel/ipu-buttress.c
index 7967fe5..beff618 100644
--- a/drivers/media/pci/intel/ipu-buttress.c
+++ b/drivers/media/pci/intel/ipu-buttress.c
@@ -708,7 +708,7 @@ ipu_buttress_add_psys_constraint(struct ipu_device *isp,
 						  b->psys_fused_freqs.max_freq);
 		ipu_buttress_set_psys_freq(isp, b->psys_min_freq);
 	}
-	mutex_unlock(&isp->buttress.cons_mutex);
+	mutex_unlock(&b->cons_mutex);
 }
 EXPORT_SYMBOL_GPL(ipu_buttress_add_psys_constraint);
 
-- 
2.7.4

